tethys_cli/gen_templates/settings

```diff
@@ -129,17 +109,14 @@ REST_FRAMEWORK = {
     )
 }
 
-AUTHENTICATION_BACKENDS = (
-#    'tethys_services.backends.hydroshare.HydroShareOAuth2',
-#    'social_core.backends.linkedin.LinkedinOAuth2',
-#    'social_core.backends.google.GoogleOAuth2',
-#    'social_core.backends.facebook.FacebookOAuth2',
-    'django.contrib.auth.backends.ModelBackend',
-    'guardian.backends.ObjectPermissionBackend',
-)
+AUTHENTICATION_BACKENDS = [
+{%- for item in backends %}
+    '{{ item }}',
+{%- endfor %}
+]
 
 # Password validation
-# https://docs.djangoproject.com/en/1.9/ref/settings/#auth-password-validators
+# https://docs.djangoproject.com/en/2.1/ref/settings/#auth-password-validators
```

Comment:
Let's change all Django docs references to point at 2.2, since we plan to upgrade to that before release.